README.md

```diff
@@ -48,7 +48,13 @@ This repo contains the components Mintlify uses in our open-source documentation
 
 ## Installation
 
-`npm install @mintlify/components`
+### npm
+
+`npm install @headlessui/react @mintlify/components`
```

Comment:
Prefer `install-peerdeps`, see https://github.com/mintlify/config/tree/main/packages/prettier-config as an example

Reply:
Yea that's nice. I'll use that. I'll do it similarly to the `config` example and keep both methods, as some people might be on different react versions or on `preact` and don't need to install all peerdependencies.